docs/examples/api-reference/operators/dedup.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 from datetime import datetime, timezone
+from fennel.dtypes import Session, Tumbling
```

Comment:
Can you plz move this into the snippet so that people can easily copy paste these imports as well?